DROD/RoomWidget.cpp

```diff
@@ -2361,18 +2364,22 @@ void CRoomWidget::RenderRoomLayers(SDL_Surface* pSurface, const bool bDrawPlayer
 	ASSERT(this->pRoom);
 
 	RenderFogInPit(pSurface);
-	this->pOLayerEffects->DrawEffectsOfType(EIMAGEOVERLAY, false, true, pSurface);
+	// @FIXME-FREEZING
```

Comment:
To address (or clarify what's the issue in the comment, and why we can defer changes till later).

Reply:
Those are not needed anymore, removed